lib/iris/tests/unit/cube/test_Cube.py

```diff
@@ -409,6 +409,160 @@ def test_kwargs(self):
         self.assertMaskedArrayEqual(expected_result, res_cube.data)
 
 
+class Testslices(tests.IrisTest):
+    '''
+    This class tests the slices capability of iris.cube.Cube
+    '''
+    def setUp(self):
+        '''
+        setup a 4D iris cube, each dimension is length 1.
+        The dimensions are;
+            dim1: height
+            dim2: time
+            dim3: longitude
+            dim4: latitude
+        '''
+        self.cube = iris.cube.Cube(np.array([[[[8.]]]]))
+        self.cube.add_dim_coord(iris.coords.DimCoord([0], "height"), [0])
+        self.cube.add_dim_coord(iris.coords.DimCoord([0], "time"), [1])
+        self.cube.add_dim_coord(iris.coords.DimCoord([0], "longitude"), [2])
+        self.cube.add_dim_coord(iris.coords.DimCoord([0], "latitude"), [3])
+
+    @staticmethod
+    def expectedcube_setup(dim1name, dim2name, dim3name):
```

Comment:
Please us snake case for function and variable names (separate each word with an underscore).